internal/crypto/envelope/enveloper/yckms/enveloper.go

```diff
@@ -46,23 +47,30 @@ func (enveloper *Enveloper) DecryptKey(encryptedKey []byte) ([]byte, error) {
 	return rsp.Plaintext, nil
 }
 
-func (enveloper *Enveloper) SerializeEncryptedKey(encryptedKey []byte) []byte {
-	return serializeEncryptedKey(encryptedKey)
+func (enveloper *Enveloper) SerializeEncryptedKey(encryptedKey []byte, keyID string) []byte {
+	return serializeEncryptedKey(encryptedKey, keyID)
 }
 
-func serializeEncryptedKey(encryptedKey []byte) []byte {
+func serializeEncryptedKey(encryptedKey []byte, keyID string) []byte {
 	/*
 		magic value "envelope-yc-kms"
 		scheme version (current version is 1)
+		uint32 - keyID len
+		keyID ...
 		uint32 - encrypted key len
 		encrypted key ...
 	*/
 
+	result := append([]byte(magic), schemeVersion)
+
+	keyIDLen := make([]byte, 4)
```

Comment:
Let's also extract this size to const like it is done [in other places](https://github.com/wal-g/wal-g/blob/953036f7482b0f33c7602aebacd9d91d0476ebf9/internal/databases/postgres/pagefile.go#L34) to get rid of the magic constants